torch/_dynamo/eval_frame.py

```diff
@@ -1221,7 +1224,9 @@ def result_capturing_wrapper(*graph_inputs):
         if (
             (shape_env := getattr(fake_mode, "shape_env", None)) is not None
             and (dim_constraints := shape_env.dim_constraints) is not None
-            and not skipfiles.check(inspect.getsourcefile(call_to_inspect))
```

Comment:
you could have just kept the old check() around now LOL

Reply:
yeah, let's just define a:
```
def check(f):
   return check_verbose(f).skipped
```
and keep the simpler API